### PR TITLE
Adding textnets 0.4.10

### PR DESCRIPTION
**Pull request by jboynyc**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jboynyc** [comment]
@conda-forge-admin, please ping team

File: recipes/textnets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  entry_points:
+    - textnets = textnets.cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - Click >=7.0
```
Comment (chrisburr) [inline recipes/textnets/meta.yaml:26]: conda-forge packages are always lowercase
```suggestion
    - click >=7.0
```

File: recipes/textnets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+test:
+  imports:
+    - textnets
+  requires:
+    - pytest
+
+about:
+  home: https://textnets.readthedocs.io
+  license: GPL-3.0-or-later
+  license_family: GPL3
```
Comment (chrisburr) [inline recipes/textnets/meta.yaml:44]: This field is now added automatically:
```suggestion
```

File: recipes/textnets/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  run:
+    - python >=3.7
+    - Click >=7.0
+    - pandas ==1.0.4
+    - cairocffi ==1.1.0
+    - python-igraph ==0.8.2
+    - spacy ==2.3.0
+    - scipy ==1.4.1
+    - toolz ==0.10.0
+    - leidenalg ==0.8.1
```
Comment (chrisburr) [inline recipes/textnets/meta.yaml:33]: Do these really need to be this restrictive?

Comment (jboynyc) [reply recipes/textnets/meta.yaml:33]: I suspect not, I just used the same versions pinned in `setup.py`. I can try to turn these into looser constraints.

Comment (jboynyc) [reply recipes/textnets/meta.yaml]: 2a06bbf makes these far less restrictive. Let me know if that looks better to you.

Comment (chrisburr) [reply recipes/textnets/meta.yaml]: Yup that looks good, I'll merge when the CI passes. Thanks! 🍰

**Pull request merged**